include/eve/function/betainc_inv.hpp

```diff
@@ -1,4 +1,3 @@
-//==================================================================================================
```

Comment:
why removing this line ?

Reply:
ces lignes sont inutiles et mettent un espacement plus grand et moche avec ce qui précéde (et n'ont jamais été partout)
si tu veux je ne les enléverais plus si j'en vois d'autres, mais pour moi ça faisait partie du nettoyage et je ne prendrai pas la peine de remettre celles que j'ai oté, parce que je ne vois pas ce que ça apporte.
Merge SVP.